pkg/sideinput/examples/simple-sideinput/udf/main.go

```diff
@@ -0,0 +1,73 @@
+package main
+
+import (
+	"context"
+	"log"
+	"os"
+	"path"
+
+	"github.com/fsnotify/fsnotify"
+
+	"github.com/numaproj/numaflow-go/pkg/mapper"
+)
+
+var sideInputPath = "/var/numaflow/side-inputs"
+var sideInputName = "myticker"
+
+func mapFn(_ context.Context, _ []string, _ mapper.Datum) mapper.Messages {
+	// Perform map operation here
+	return mapper.MessagesBuilder().Append(mapper.NewMessage([]byte("test_value")))
```

Comment:
```suggestion
	return mapper.MessagesBuilder().Append(mapper.NewMessage([]byte("some value")))
```

word `test` has a bad connotation :)